### DISCUSSION TITLE
¿Cómo añado la red de Polygon a mi Metamask?

### DISCUSSION THREAD

**Discussion by IBaixauli** [NGDAOorg/lapalma#6] [Preguntas y respuestas] [answered]
¿Cómo añado la red de Polygon a mi Metamask?

**Comment by ruth-ga** [comment, marked as answer]
Después de reiniciar el sistema, apagar el ordenador... actualizar muchas veces la página...probar varios enlaces que había usado antes. IBaixauli, me pasó este enlace  que resultó ser totalmente eficaz. 
https://docs.polygon.technology/docs/develop/metamask/config-polygon-on-metamask/#add-the-polygon-network-manually
clicando y siguiendo el sistema manual de configuración.

**Comment by mreyesFL** [comment]
Voy a intentarlo con el enlace que has compartido. Muchas gracias.

**Comment by Miller1985** [comment]
Recordar que en el campo de url, si no os funciona la que aparece en la imagen; podeis probar con estas que os pongo aqui:
 a) https://rpc-mainnet.matic.network 
b) https://rpc-mainnet.maticvigil.com 
c) https://rpc-mainnet.matic.quiknode.pro 
d) https: // matic-mainnet .chainstacklabs.com 
e) https://matic-mainnet-full-rpc.bwarelabs.com
f) https://matic-mainnet-archive-rpc.bwarelabs.com

**Reply by Miller1985** [reply]
https: // matic-mainnet.chainstacklabs.com
Esta era erronea ahora creo que se podra utilizar
